packages/inputs-tiny/src/select/index.jsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { SIZES_KEYS } from '@/src/constants';
+import Meta from '../textfields_meta';
+import styles from './index.module.scss';
+
+const Select = ({
+  label,
+  error,
+  value = '',
+  options,
+  size = 'medium',
+  onChange,
+  className,
+  disableBorder,
+  disabled,
+  ...rest
+}) => {
+  const handleChange = (e) => {
+    const val = e.target.value;
+
+    onChange(val, e);
+  };
+
+  return (
+    <div>
+      <div
+        className={clsx(
+          styles.inputWrapper,
+          styles[size],
+          {
+            [styles.disableBorder]: disableBorder,
+            [styles.withError]: error,
+            [styles.disabled]: disabled,
+          },
+        )}
+      >
+        <label
+          className={clsx(styles.label)}
+        >
+          <span
+            className={clsx(
+              styles.labelText,
+              styles[size],
+            )}
+          >
+            {label}
+          </span>
+
+          <select
+            className={clsx(styles.input, className)}
+            onChange={handleChange}
+            value={value}
+            disabled={disabled}
+            {...rest}
+          >
+            {options.map((option) => (
+              <option
+                key={option.value}
+                value={option.value}
+              >
+                {option.key}
```

Comment:
rather call it `label`? just think if we would want to pass jsx here 

maybe we would want a different api in that case? 

- https://vuetifyjs.com/en/components/selects/#selection vuetify does it like you 
- https://mui.com/components/selects/ mui is very different